bedrock/test.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+var bedrock = require('bedrock');
+require('../lib/main.js');
```

Comment:
We're using `index` not `main` for node.js (it's their convention. We use `main` for bower modules (it's their convention).